sycl/doc/extensions/supported/sycl_ext_intel_device_info.md

```diff
@@ -345,5 +345,5 @@ An invalid object runtime error will be thrown if the device does not support as
 Then the maximum memory bandwidth can be obtained using the standard get\_info() interface.
 
     if (dev.has(aspect::ext_intel_max_mem_bandwidth)) {
-      auto maxBW = dev.get_info<info::device::ext_intel_max_mem_bandwidth>();
+      auto maxBW = dev.get_info<ext::intel::info::device::max_mem_bandwidth>();
     }
```

Comment:
Since this extension is "supported", we really should document the deprecated queries.  I'd suggest a new section at the bottom (a peer of "Maximum Memory Bandwidth") named something like "Deprecated queries".  That section can list all of the deprecated info descriptors and their equivalent new names.